torchx/Makefile

```diff
@@ -22,7 +22,11 @@ check_libtorch:
 $(TORCHX_SO): c_src/torchx.cpp c_src/nx_nif_utils.hpp
 	@ mkdir -p $(PRIV_DIR)
 	@ mkdir -p $(CMAKE_BUILD_DIR)
-	@ ln -sf $(abspath $(LIBTORCH_DIR)/lib) $(PRIV_DIR)/libtorch
+	@ if [ "${BUILD_EMBEDDED}" = "false" ]; then \
+		ln -sf $(abspath $(LIBTORCH_DIR)/lib) $(PRIV_DIR)/libtorch ; \
+	else \
+		cp -a $(abspath $(LIBTORCH_DIR)/lib) $(PRIV_DIR)/libtorch ; \
+	fi
```

Comment:
I'd invert the test to validate if `"${BUILD_EMBEDDED}" = "true"`, which makes my comment on `"nil"` below kinda superfluous

Reply:
OK, I've inverted the test